### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by marijanbrvar**
Add Jest to an existing configuration.
Add test file for test class where we ware testing Add task, Task equality, and Remove charge.

### CODE REVIEW

File: src/_test_/task.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    expect(res.tasks).toStrictEqual(expected);
+  });
+  test('Remove Task', () => {
+    const expected = [
+      { index: 0, description: "Don't forget Cigars!", completed: false },
+    ];
+    const res = new Task();
+    res.deleteTask(1);
+    expect(res.tasks).toStrictEqual(expected);
+  });
```
Comment (Davidosky007) [inline src/_test_/task.test.js]: * I suggest you use mocks for these functions which updates the local storage and manipulates the DOM [Refer to the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md#:~:text=Some%20of%20those,in%20the%20DOM.) as it is required for this project

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @marijanbrvar @2bleO 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly endeavour to write tests for all functions responsible for updating local storage and manipulating the DOM as stated in the requirements of the project.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/_test_/task.test.js (hunk 1 above)
Comment (DeliceLydia) [reply src/_test_/task.test.js]: - [x] This was not addressed as requested by the previous reviewer please use mock for functions that used localStorage, DOM and submit another CRR

**Review by carlos-ssh — APPROVED** [review]
Hi @marijanbrvar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**